xml/deployment_register.xml

```diff
@@ -584,4 +584,60 @@ https://www.suse.com/products/server/features/modules.html</screen>
 
   </sect2>
  </sect1>
+ <sect1 xml:id="sec-register-sle-suseconnect-timer">
+  <title>&suseconnect; keep-alive timer</title>
+  <para>
+   From version 0.3.33, the &suseconnect; package ships with two &systemd; units:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     <filename>suseconnect-keepalive.service</filename>: a one-shot service
```

Comment:
If you wouldn't mind, please change "one-shot" to "one-time". (As a US resident, I'm a bit sensitive to anything shooting-related). Thank you!

Reply:
Thank you for the feedback, and for the learning, I never thought about the term before :)